app/models/contributor.rb

```diff
@@ -112,6 +112,15 @@ def active?
   end
   alias active active?
 
+  def avatar_url=(url)
+    begin
+      remote_file_location = URI(url)
+    rescue URI::InvalidURIError, ArgumentError
```

Comment:
You’re handling the `InvalidURIError` in case `url` is `nil` (or similar), e.g. in case the contributor has no profile picture set, right?

What’s the case for the `ArgumentError`?

Reply:
`InvalidURIError` handles "lksjdflsjdfl" and `ArgumentError` handles `nil`.

Reply:
I believe a timeout would occur on line 121. Do you want me to test this?

Reply:
Regarding the use of `rescue` in order to handle `nil`, I think I’d have preferred testing for that explicitly. That would have made the intent clearer for me, e.g.

```ruby
return unless url
```

But that’s fine, I think. I’ll approve and you can decide if you want to change anything else.

Reply:
Makes sense, :+1: for clear intention.